### ISSUE TITLE
Hero image file path not updating via .md

### ISSUE THREAD

**Issue by benrito** [theirc/airbel#43]
Hey @tbeck , can you give us the 60 second explanation of Cloudvent CMS upload + file compression?

We were trying to manually update hero images via md file, but the hero image was defaulting to background-image: url(https://via.placeholder.com/1800x500?text=Hero+image).

I then noticed that when we upload from the CMS, we get something like url(//d1qmdf3vop2l07.cloudfront.net/icy-ship.cloudvent.net/compressed/38062aa8a850a47eb6d8d652acbc49b7.jpg).

This is obviously really great for optimization, but think it also means that we have certain CMS hooks in here and should avoid linking project assets like this directly in the repo. Is that right?

**Comment by tbeck** [comment]
Hi @benrito - I'll do my best.

**Fallback hero image**
In the hero.html include we have a fallback to give a visual cue when the hero_image field has not been filled in for a project: https://github.com/ocupop/airbel/blob/master/src/_includes/project/hero.html

`{{ project.hero_image | default: site.placeholders.hero }}`

**CDN / URL Paths**
Using the **Coach Everu** project as an example: [raw](https://raw.githubusercontent.com/ocupop/airbel/master/src/_projects/coach-everu.md)

The hero_image: is set to a relative path `/uploads/dscf1201-1.jpg`

Cloudcannon is currently configured to optimize and serve assets from their CDN. This happens during build and does not have an effect on the paths used within Jekyll's src folder. During the build they replace relative paths with absolute urls pointing to their CDN. [[documentation](https://docs.cloudcannon.com/building/optimisations/)]

This results in the following:
```
<div class="bg-image aspect-wide"
        style="background-image: url(//d1qmdf3vop2l07.cloudfront.net/icy-ship.cloudvent.net/compressed/38062aa8a850a47eb6d8d652acbc49b7.jpg)"
       data-cms-original-style="background-image: url(/uploads/dscf1201-1.jpg)"></div>
```

Note the `data-cms-original-style` attribute.

In short... we should be able to link to files within the repo using relative paths. If you decide to move away from Cloudcannon. ie... Github Pages, nothing will change, but the build process.

Hopefully this is an okay explanation, but hit me up if you want to jump on a quick call and step through any of the details.
